src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_NUMBER = 0;
+    private static final int OPERATION_VALUE_NUMBER = 1;
+    private static final int SUPPLY_SUM_INDEX = 0;
+    private static final int BUY_SUM_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(writeReport(calculateSums(readFile(fromFileName))), toFileName);
```

Comment:
this expression is overloaded and difficult to read, please introduce some varialbles